### ISSUE TITLE
[Request] While watching video, add tags that do not create markers

### ISSUE THREAD

**Issue by thegdthadman** [fupdec/Adult-Video-Database#53]
Being able to add scene markers while viewing a video is useful when time-stamped bookmarks are desired. Sometimes, however, watching a video will make me think of a tag that would apply to the video but is not necessary to create a marker. Would like the ability to add tags while watching video, but not have that action create a marker.

**Comment by fupdec** [comment]
Yes, that's a good suggestion. I'm just working on adding tags directly from the performer edit dialog. #48
